package.json

```diff
@@ -227,5 +227,9 @@
   },
   "msw": {
     "workerDirectory": "public"
+  },
+  "dependencies": {
+    "@types/react-select": "^5.0.1",
+    "react-select": "^5.7.0"
```

Comment:
This increases final bundle size by 5kb post-gzip. This is far too much in my opinion. If we manage to resolve all the other issues with this dropdown, it has to be lazy-loaded.